nautobot/docs/plugins/index.md

```diff
@@ -8,7 +8,7 @@ The Nautobot plugin architecture allows for plugins to do any or all of the foll
 
 ### Extend the existing Nautobot UI
 
-* **Add content to existing model detail views.** A plugin can inject custom HTML content within the view of a core Nautobot model. This content can appear in the left column, right column, or full width of the page, and can also include custom buttons at the top of the page.
+* **Add content to existing model detail views.** A plugin can inject custom HTML content within the view of a core Nautobot model. This content can appear in the left column, right column, or full width of the page, can include custom buttons at the top of the page and can include extra tabs which appear at the end of object detail tabs list.
```

Comment:
Since this is the newest and arguably most complicated feature, can you please move this bullet to the bottom of this list?